### PR TITLE
Issue #7595: Update doc UniqueProperties

### PR DESCRIPTION
**Pull request by ImmortalRabbit**
Issue: #7595 

Screen Shot 2020-04-21 at 18 21 26

`$ cat config.xml`
```
<?xml version="1.0"?>
<!DOCTYPE module PUBLIC
          "-//Checkstyle//DTD Checkstyle Configuration 1.3//EN"
          "https://checkstyle.org/dtds/configuration_1_3.dtd">
<module name="Checker">
    <module name="UniqueProperties">
      <property name="fileExtensions" value="properties"/>
    </module>
</module>
```
`java -Djava.util.logging.config.file=log.properties -jar checkstyle-8.30-all.jar -c config.xml java_sandbox/src/com/codestyle/sandbox/Test.java`

```
Starting audit...
[ERROR] /Users/demezhanmarikov/IdeaProjects/java_sandbox/src/com/codestyle/sandbox/Properties.properties:2: Duplicated property 'key' (2 occurrence(s)). [UniqueProperties]
Audit done.
Checkstyle ends with 1 errors.
```

### CODE REVIEW

File: src/main/java/com/puppycrawl/tools/checkstyle/checks/UniquePropertiesCheck.java [hunk 1]
```diff
@@ -58,7 +58,17 @@
...
+ * <p>
+ * Example:
+ * </p>
+ * <p>
+ * File name: main.properties
+ * </p>
+ * <pre>
+ * host=localhost // OK
+ * key=value1 // Violation - duplicate property 'key'
+ * key=value2 // Violation - duplicate property 'key'
```
Comment (strkkk) [inline src/main/java/com/puppycrawl/tools/checkstyle/checks/UniquePropertiesCheck.java:70]: 1. Please leave violation comment only on line where it is reported.
2. Violation -> violation
3. Exaplanation of violation should be with comma, e.g. `violation, here is explanation`

Comment (strkkk) [inline src/main/java/com/puppycrawl/tools/checkstyle/checks/UniquePropertiesCheck.java:70]: Please also update example for violation/OK with keys with dots, like `host.port`

**Comment by romani** [comment]
@ImmortalRabbit , ping

**Comment by strkkk** [comment]
Closed as abandoned

**Pull request closed without merging**